lib/index.ts

```diff
@@ -120,15 +149,13 @@ function getPine(param: {
 declare namespace getPine {
```

Comment:
You can change to `export function getPine` above and remove the need to do this declare namespace - the only reason we've got it was to be able to convert to typescript with no breaking changes, since we're adding breaking changes now it makes sense to improve the exports imo